interpreter.go

```diff
@@ -360,6 +360,21 @@ func (i *interpreter) objectIndex(loc *LocationRange, obj value, f string, offse
 	}
 }
 
+func (i *interpreter) evalAdd(leftVal, rightVal value) (value, error) {
+	switch leftVal := leftVal.(type) {
+	case *valueNumber:
+		left := leftVal.value
+		right := rightVal.(*valueNumber).value
+		return makeValueNumber(left + right), nil
+	case *valueString:
+		left := leftVal.value
+		right := rightVal.(*valueString).value
+		return makeValueString(left + right), nil
```

Comment:
I think a `default` is preferable to completing the `switch` without hitting a `case`.